SubEthaEdit-Mac/Source/TCMMillionMonkeys/TCMMMSession.m

```diff
@@ -206,14 +192,12 @@ - (TCMMMLoggingState *)loggingState {
 - (void)setLoggingState:(TCMMMLoggingState *)aState {
 	if (aState != I_loggingState) {
 		[I_loggingState setMMSession:nil];
-		[I_loggingState autorelease];
```

Comment:
this probably needs the __ autoreleasing. And as we do something with the previous value, it isn't even ugly:
```objc
__autoreleasing typeof(I_loggingState) old = I_loggingState;
[old setMMSession:nil];
I_loggingState = aState;
[I_loggingState setMMSession:self];
```

Reply:
As almost all Code is now ARC, imo this should not be relevant anymore